README.md

```diff
@@ -3,10 +3,10 @@
 Responsive header for FT branded sites. See the [Origami Navigation Service](https://www.ft.com/__origami/service/navigation) to populate `o-header` markup with real navigation data.
 
 - [Markup](#markup)
-- [JavaScript](#JavaScript)
+- [JavaScript](#javascript)
 - [Sass](#sass)
 - [Browser Support](#browser-support)
-- [Migration Guide](#migration)
+- [Migration guide](#migration-guide)
```

Comment:
maybe we should be using a title case? Just spotted `Browser Support` which does?

Reply:
ah yeah